lib/utils/patterns.ts

```diff
@@ -77,16 +78,28 @@ export function authenticate(options: {}): Bluebird<void> {
 		});
 }
 
-export async function exitIfNotLoggedIn(): Promise<void> {
+export async function checkLoggedIn(): Promise<void> {
 	const balena = getBalenaSdk();
 	if (!(await balena.auth.isLoggedIn())) {
-		exitWithExpectedError(stripIndent`
+		throw new NotLoggedInError(stripIndent`
```

Comment:
`NotLoggedInError` is an instance of `ExpectedError`, so it uses the new error handling capability that avoids reporting the error to Sentry.io while also avoiding the premature call to `process.exit()`. I am calling `checkLoggedIn()` (instead of `exitIfNotLoggedIn()`) in another PR I'm working on.